projects/plugins/jetpack/to-test.md

```diff
@@ -1,87 +1,39 @@
-## 10.6
+## 10.7
 
 ### Before you start
 
 - **At any point during your testing, remember to [check your browser's JavaScript console](https://codex.wordpress.org/Using_Your_Browser_to_Diagnose_JavaScript_Errors#Step_3:_Diagnosis) and see if there are any errors reported by Jetpack there.**
```

Comment:
Proofing the testing instructions, everything looks ok to me on the surface. However it might make sense to update this link to https://wordpress.org/support/article/using-your-browser-to-diagnose-javascript-errors/#step-3-diagnosis